src/WebJobs.Script.WebHost/Security/IScriptSecretSerializer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    internal interface IScriptSecretSerializer
```

Comment:
nit: might as well be consistent with visibility. I think across the board we've made everything in WebHost public, since it isn't a class lib? Same for all your other new types.